### DISCUSSION TITLE
I keep getting this error when trying to install dependencies

### DISCUSSION THREAD

**Discussion by UltraHydro** [catsout/wallpaper-engine-kde-plugin#314] [Q&A]
![image](https://github.com/catsout/wallpaper-engine-kde-plugin/assets/123855600/dd0654db-e851-4eed-ba66-76edee11d354)

**Comment by UltraHydro** [comment]
It does this for everything it downloads

**Comment by Impqxr** [comment]


**Reply by UltraHydro** [reply]
I still get the same error after doing this
